packages/core/src/Stores/notification-store.js

```diff
@@ -1130,6 +1135,16 @@ export default class NotificationStore extends BaseStore {
                 type: 'announce',
                 should_hide_close_btn: false,
             },
+            poa_older_than_six_months: {
+                key: 'poa_older_than_six_months',
+                header: localize('Lets get your address verified'),
+                message: localize('Please submit your proof of address.'),
```

Comment:
lets use Localize component instead of localize

Reply:
Done